quivr/tables.py

```diff
@@ -1116,7 +1140,7 @@ def _encode_attr_dict(cls, attrs: dict[str, Any]) -> dict[bytes, bytes]:
             result[k.encode("utf8")] = descriptor.to_bytes(pytyped)
         return result
 
-    def apply_mask(self, mask: pa.BooleanArray | np.ndarray[bool, Any] | list[bool]) -> Self:
+    def apply_mask(self, mask: pa.BooleanArray | npt.NDArray[np.bool_] | list[bool]) -> Self:
```

Comment:
Nice, there are already tests for these three types: https://github.com/B612-Asteroid-Institute/quivr/blob/f7c92ae276389000c66b1a29907f497faf3a2003/test/test_tables.py#L1051-L1080